Sources/IO/Core/HttpDataSetReader/index.js

```diff
@@ -247,6 +238,29 @@ function vtkHttpDataSetReader(publicAPI, model) {
     return publicAPI.updateMetadata(!!options.loadData);
   };
 
+  publicAPI.parseObject = (
+    manifest,
+    { loadData, baseUrl, deepCopy = true }
+  ) => {
+    if (baseUrl) {
+      model.baseURL = baseUrl;
+    }
+
+    const dataset = deepCopy ? structuredClone(manifest) : manifest;
```

Comment:
I have a doubt about the structuredClone call... This is quite a new recent feature and I don't know if it's supported enough to have it into the VTK.js code base : https://developer.mozilla.org/en-US/docs/Web/API/structuredClone#browser_compatibility
@jourdain @floryst WDYT?

Reply:
I'm not sure what's that for. What is the use case of that?

Reply:
@jourdain deep copies the dataset manifest, as to not mutate it when it gets processed.

@Julesdevops I'd check to see if babel polyfills it with core-js. If not, we may need to bring in the polyfill, or use a different deep-copy.

Reply:
It is indeed polyfilled in core-js: https://github.com/zloirock/core-js#structuredclone
Thanks for feedback